### ISSUE TITLE
Slimbox2 creates "Empty Links," which violates the WCAG 2.0 Accessibility Guidelines

### ISSUE THREAD

**Issue by SunnySanDiego** [cbeyls/slimbox#53]
[ WCAG 2.0 Guideline 2.4.4 "Link Purpose (In Context)"](https://webaim.org/standards/wcag/checklist#sc2.4.4) states:

> The purpose of each link (or form image button or image map hotspot) can be determined from the link text alone, or from the link text and its context (e.g., surrounding text, list item, table cell, or table headers).

In short, this means that an HTML document should not have hyperlinks with no content. The [WAVE Accessibility Tool](http://wave.webaim.org/) reports three such violations in Slimbox2, stating:

> **Why It Matters** If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users.

> **How to Fix It** Remove the empty link or provide text within the link that describes the functionality and/or target of that link.

The three empty links created by Slimbox2 are:

- lbPrevLink
- lbNextLink
- lbCloseLink

Thanks
